README.md

```diff
@@ -55,8 +55,7 @@ This repo uses [coreos-assembler repo](https://github.com/coreos/coreos-assemble
 The test downloads the latest RHCOS image and runs network related tests, using the local `setup-ovs.sh` script.
 You can run these tests manually on Fedora by running the test script:
 ```bash
-sudo dnf install -y git go make wget qemu qemu-img swtpm
-./tests/test-coreos.sh
+sudo ./tests/setup.sh && ./tests/test-coreos.sh
```

Comment:
Could you keep it on two lines? So the first time I'm following this README I triple click both lines. And all the other times I can triple click only one of them. With this single-line command, I'd have to select it specifically

Reply:
sure, [Done](https://github.com/RHsyseng/rhcos-slb/compare/6b9a9f4a537654eea7c9a24283c451c894ba50ad..7e325e30c347c14cd1f8d816959b243c44d4f6f2)